lib/sampleNode/src/locales/en-US/default.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
The file's name seems meaningless.
There is some standard about the file structure? If not, I suggest something like:
`src/i18n-locales/en-US.json` or  `src/i18n/en-US.json`